__tests__/setup-go.test.ts

```diff
@@ -49,6 +50,8 @@ describe('setup-go', () => {
     inSpy.mockImplementation(name => inputs[name]);
     getBooleanInputSpy = jest.spyOn(core, 'getBooleanInput');
     getBooleanInputSpy.mockImplementation(name => inputs[name]);
+    exportVarSpy = jest.spyOn(core, 'exportVariable');
+    extractTarSpy = jest.spyOn(core, 'exportVariable');
```

Comment:
is this a copy/paste error? Both are `exportVariable`, extractTarSpy should be `extractTar`.